optimum/intel/neural_compressor/utils.py

```diff
@@ -30,6 +30,8 @@
 
 
 CONFIG_NAME = "best_configure.yaml"
+ENGINE_MODEL_NAME = "model.bin"
+ENGINE_MODEL_CONFIG = "conf.yaml"
```

Comment:
not used so might make sense to  remove this for now 
```suggestion
```

Reply:
removed